src/synopsys-action/tools-parameter.ts

```diff
@@ -104,21 +113,41 @@ export class SynopsysToolsParameter {
     return command
   }
 
-  getFormattedCommandForCoverity(userName: string, passWord: string, coverityUrl: string, projectName: string): string {
-    validateCoverityParams(userName, passWord, coverityUrl, projectName)
+  getFormattedCommandForCoverity(userName: string, passWord: string, coverityUrl: string, projectName: string, streamName: string, installDir: string, policyView: string, repositoryName: string, branchName: string): string {
+    validateCoverityParams(userName, passWord, coverityUrl, projectName, streamName)
     const covData: InputData<Coverity> = {
       data: {
         coverity: {
           connect: {
             user: {name: userName, password: passWord},
-            url: coverityUrl,
-            project: {name: projectName},
-            policy: {view: 'SAST'}
-          }
-        }
+            url: coverityUrl
+          },
+          project: {name: projectName},
+          stream: {name: streamName}
+        },
+        project: {}
+      }
+    }
+
+    const osName = process.platform
+    if (osName === 'win32') {
+      if (installDir) {
```

Comment:
Is it like the installDir can be passed only in case of windows?
And if it is mandatory for windows, then we can expect this input as a mandatory input by putting validation for it's availability. And for other OS, it can be optional

Reply:
added regex to verify windows path..